rock_paper_scissors/api/models.py

```diff
@@ -0,0 +1,56 @@
+from sqlalchemy import Column, Integer, String
```

Comment:
The lines beginning with `from sqlalchemy import ` should be merged into a single import statement:
```python
from sqlalchemy import (
    Column,
    Enum,
    ForeignKey
    Integer,
    String
)
```